maps/nerva/nerva_define.dm

```diff
@@ -4,6 +4,16 @@
 	path = "nerva"
 
 	lobby_screens = list('maps/nerva/nerva_lobby.dmi')
+	lobby_tracks = list(
+		/singleton/audio/track/df_theme,
+		/singleton/audio/track/digit_one,
+		/singleton/audio/track/dilbert,
+		/singleton/audio/track/chasing_time,
+		/singleton/audio/track/human,
+		/singleton/audio/track/lysendraa,
+		/singleton/audio/track/rimward_cruise,
```

Comment:
```suggestion
		/singleton/audio/track/human,
		/singleton/audio/track/level3_mod,
		/singleton/audio/track/lysendraa,
		/singleton/audio/track/rimward_cruise,
```
Just realized this morning there's another classic missing (level3_mod is the goofy jazzy track). I can live without dilbert, but level3_mod is a hill I will happily die on.